user_docs/en/changes.md

```diff
@@ -41,6 +41,7 @@ Unicode CLDR has also been updated.
 * NVDA will now report figures with no accessible children, but with a label or description. (#14514)
 * When reading by line in browse mode, "caption" is no longer reported on each line of a long figure or table caption. (#14874)
 * In the Python console, the last unexecuted command will no longer be lost when moving in the input history. (#16653, @CyrilleB79)
+* A unique anonymous ID is now sent as part of NVDA usage statistics gathering. (#16266)

```

Comment:
Minor suggestion: Clarify that this is optional (could alternatively leave the start of the line the same and say "optional NVDA usage statistics gathering" at the end)

```suggestion
* An optional, unique, anonymous ID is now sent as part of NVDA usage statistics gathering. (#16266)

```

Reply:
```suggestion
* A unique anonymous ID is now sent as part of optional NVDA usage statistics gathering. (#16266)
```